### ISSUE TITLE
⌨ Unable to select classification level

### ISSUE THREAD

**Issue by BrookeCameron** [GCTC-NTGC/gc-digital-talent#13391]
### ⌨ Accessibility and Expected Behaviour

Unable to tab or use arrows to select a value for the job classification level under Government employee information. 

### 📋 Steps to Reproduce

1. Navigate to Personal information 
2. fill in government employee information 
3. attempt to tab through the classification level

### 🕵️ Details

this seems to be an issue with VoiceOver specifically. 

The only way the user was able to input this information was by manually typing in the number. 

### What operating systems are you encountering the problem on?

macOS

### What browsers are you encountering the problem on?

Firefox

### What assistive technologies are you encountering the problem on?

VoiceOver

### 📸 Screenshot

https://github.com/user-attachments/assets/4ca96d07-b894-4c7b-b955-31c827c28109

**Comment by petertgiles** [comment]
@BrookeCameron I haven't been able to reproduce this in Firefox on Linux.  Can you reproduce this in Firefox on Mac OS with Voiceover?

**Comment by mnigh** [comment]
@BrookeCameron I wasn't able to reproduce this behaviour on Firefox on macOS using VoiceOver.

## Screen recording
https://github.com/user-attachments/assets/b21e2fbf-dd49-4b2e-8d23-2e4fd271eee3

How about @esizer or @brindasasi, are either of you able to reproduce on Firefox on macOS using VoiceOver?

**Comment by esizer** [comment]
> How about [@esizer](https://github.com/esizer) or [@brindasasi](https://github.com/brindasasi), are either of you able to reproduce on Firefox on macOS using VoiceOver?

I'm joining the "Cannot reproduce" party.

**Comment by petertgiles** [comment]
Closing for now.  Feel free to reopen if it gets reported again or we're able to reproduce.
